README.md

```diff
@@ -63,8 +63,9 @@ programs using Rust. It includes:
 
 `LLVM` is required in your build system to compile BPF bytecode using RedBPF.
 
-- **LLVM 13**  
-  It is needed to compile BPF bytecode.
+- **LLVM 14 or 13**
+  It is needed to compile BPF bytecode. If you're using Rust 1.60 or later, LLVM 14 is required.
+  If you need to use LLVM 13, use the feature `llvm13-0`.
```

Comment:
The feature's name is `llvm13` in `Cargo.toml`.

Reply:
Good catch, thanks!